extension/services/deezer/player.js

```diff
@@ -0,0 +1,52 @@
+/* globals helpers */
+
+this.player = (function() {
+  const SEARCH_PLAY = ".datagrid-row song a[role='button']";
+
+  class Player extends helpers.Runner {
+    action_play() {
+      const button = this.querySelector("button[aria-label='Play']");
```

Comment:
I realize spotify works like this currently, but if it's possible to not use the labels that would be better: then if someone is using a localized version of the site, these selectors will still work.